exercises/concept/library-fees/test/library_fees_test.exs

```diff
@@ -76,6 +76,12 @@ defmodule LibraryFeesTest do
       result = LibraryFees.days_late(~D[2020-06-12], ~N[2020-06-21 16:00:00Z])
       assert result == 9
     end
+
+    @tag task_id: 4
+    test "a new day starts at midnight" do
+      result = LibraryFees.days_late(~D[2020-06-12], ~N[2020-06-13 00:00:00Z])
+      assert result == 1
+    end
```

Comment:
What about this, to really drive home the point.
```elixir
    test "a new day starts at midnight" do
      result = [
        LibraryFees.days_late(~D[2020-06-12], ~N[2020-06-13 23:59:59Z]),
        LibraryFees.days_late(~D[2020-06-12], ~N[2020-06-14 00:00:00Z]),
      ]
      assert result == [1, 2]
    end
```

Reply:
sure, but as separate assertions

Reply:
Looks good to me!